external/ag-website-shared/src/components/trial-licence-form/TrialLicenceForm.module.scss

```
@@ -0,0 +1,93 @@
+@use 'design-system' as *;
+
+$trial-licence-small: 680px;
+
+.trailForm {
```

Think this one was me 😅 

```suggestion
.trialForm {
```